homeassistant/components/tfl/config_flow.py

```diff
@@ -0,0 +1,284 @@
+"""Config flow for Transport for London integration."""
+
+from __future__ import annotations
+
+from copy import deepcopy
+import logging
+from typing import Any
+from urllib.error import HTTPError, URLError
+
+from tflwrapper import stopPoint
+import voluptuous as vol
+
+from homeassistant import config_entries
+from homeassistant.config_entries import (
+    ConfigEntry,
+    ConfigFlow,
+    ConfigFlowResult,
+    OptionsFlow,
+)
+from homeassistant.core import HomeAssistant, callback
+from homeassistant.exceptions import HomeAssistantError
+import homeassistant.helpers.config_validation as cv
+from homeassistant.helpers.selector import TextSelector, TextSelectorConfig
+
+from .const import CONF_API_APP_KEY, CONF_STOP_POINTS, DOMAIN
+
+_LOGGER = logging.getLogger(__name__)
+
+STEP_USER_DATA_SCHEMA = vol.Schema(
+    {
+        vol.Optional(CONF_API_APP_KEY, default=""): cv.string,
+    }
+)
+STEP_STOP_POINT_DATA_SCHEMA = vol.Schema(
```

Comment:
Use plural if this schema allows the input of multiple stop points
```suggestion
STEP_STOP_POINTS_DATA_SCHEMA = vol.Schema(
```